packages/chakra-ui/src/SelectWidget/SelectWidget.tsx

```diff
@@ -20,20 +16,13 @@ const SelectWidget = (props: WidgetProps) => {
     disabled,
     readonly,
     value,
-    autofocus,
     onChange,
-    onBlur,
-    onFocus,
```

Comment:
Why did you remove these three props and where they were used?

Reply:
Good point,
They were only used for single-choice `Select` widget from chakra, but not for multiple-choice `Select` widget from `chakra-react-select` which made me think these props are not supported by latter. 

However `onFocus` and `onBlur` are ([as of January](https://github.com/csandman/chakra-react-select/commit/449d5b3bbc80ce14533f55b3c75402db185de844)), so I've added them back in [f5339b2](https://github.com/rjsf-team/react-jsonschema-form/pull/3182/commits/f5339b263e72d9865c7a397d4a5d19b000bf7fb4).

Reply:
I've also added back the `autoFocus` prop as of [ea34f34](https://github.com/rjsf-team/react-jsonschema-form/pull/3182/commits/ea34f349af260cdb3377bc45d60a2cec8103dba5)